sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebSecurityConfigurerAdapter.java

```diff
@@ -81,7 +77,7 @@ protected OAuth2AuthorizationRequestResolver requestResolver() {
         return new AADOAuth2AuthorizationRequestResolver(this.repo);
     }
 
-    protected AuthenticationFailureHandler failureHandler() {
-        return new AADAuthenticationFailureHandler();
+    protected AADHandleConditionalAccessFilter handleConditionalAccessFilter() {
```

Comment:
Can we delete this method? Just use `new AADHandleConditionalAccessFilter()` instead.